sound/soc/sof/trace.c

```diff
@@ -480,7 +480,8 @@ int snd_sof_init_trace(struct snd_sof_dev *sdev)
 		goto table_err;
 
 	sdev->dma_trace_pages = ret;
-	dev_dbg(sdev->dev, "dma_trace_pages: %d\n", sdev->dma_trace_pages);
+	dev_dbg(sdev->dev, "%s: dma_trace_pages: %d\n",
+		__func__, sdev->dma_trace_pages);
```

Comment:
I checked the SOF directory and all of the above are unique debug messages. Do you really think we need the function name to locate them?

Reply:
I found the messages mostly useless and the function names very useful but I guess it depends on your expertise and what you're looking for. Having to (repeatedly?) `git grep` every log line is not very productive.

The one on line 420 is especially meaningless.

Reply:
I don't disagree with @marc-hb, doing a grep on e.g. period_bytes isn't great really.